resources/views/welcome.blade.php

```diff
@@ -72,7 +72,7 @@
                     @else
                         <a href="{{ route('login') }}">Login</a>
 
-                        @if (Route::has('register'))
+                        @if (Auth::check())
```

Comment:
using `@auth` here is better or even better, since we only want superadmin and admin to be able to do this, use `@role('superadmin|admin')`